src/main/g8/src/it/scala/$package$/application/$if(add_caliban_endpoint.truthy)$SubsciberSpec.scala$endif$

```diff
@@ -0,0 +1,45 @@
+package com.example.application
+
+import com.example.ITSpec.ITSpec
+import com.example.domain._
+import com.example.infrastructure.flyway.FlywayProvider
+import zio.test.Assertion._
+import zio.test.TestAspect._
+import zio.test.{ suite, testM, _ }
+
+object deleteSpecForTests extends ITSpec(Some("items")) {
```

Comment:
This should be called `SubscriberSpec` like the file suggests.

Reply:
Right,  that was an old name, I had to oversee that one.